### PR TITLE
RT-942-Migrate-charts-on-fhir-jobs-to-new-Jenkins-server

### PR DESCRIPTION
**Pull request by amolht**
## Overview

- Update the deploy command in project.json file.

## How it was tested

- Not tested locally.

## Checklist

- [X] The title contains a short meaningful summary
- [X] I have added a link to this PR in the Jira issue
- [X] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [ ] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

**Review by vishal-elimu — DISMISSED** [review]

**Review by semicolin — DISMISSED** [review]

File: apps/cardio-patient/src/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Charts-on-FHIR Cardiovascular Health</title>
+    <title>Charts-on-FHIR Cardio Patient</title>
```
Comment (semicolin) [inline apps/cardio-patient/src/index.html:5]: Let's keep the original app title here

File: apps/cardio/src/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Charts-on-FHIR Cardiovascular Health</title>
+    <title>Charts-on-FHIR Cardio</title>
```
Comment (semicolin) [inline apps/cardio/src/index.html:5]: Let's keep the original app title here

**Review by semicolin — CHANGES_REQUESTED** [review]
Looks good, aside from the html title changes

File: apps/cardio-patient/src/index.html (hunk 1 above)
Comment (amolht) [reply apps/cardio-patient/src/index.html:5]: Can we use "Cardiovascular Patient App" as the title for cardio-patient?

**Review by semicolin — APPROVED** [review]

**Pull request merged**